### PR TITLE
Doctrine Mongo ODM Module 0.8.2 release

### PR DESCRIPTION
**Pull request by gianarb**
https://github.com/doctrine/DoctrineMongoODMModule/releases/tag/0.8.2

### CODE REVIEW

File: site/2015/03/31/doctrine_mongo_odm_module_release_0_8_2.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Doctrine Mongo ODM Module release 0.8.2
+====================================
```
Comment (Ocramius) [inline site/2015/03/31/doctrine_mongo_odm_module_release_0_8_2.rst]: Note that in RST, you should have the same amount of `=` as the length of the title

File: site/2015/03/31/doctrine_mongo_odm_module_release_0_8_2.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+The **Zend Framework Integration Team** is happy to announce the new release of **DoctrineMongoODMModule**.  
+DoctrineMongoODMModule `0.8.2` will be the last bugfix version with support for **DoctrineModule** `0.8`, and in consequence, it is the last version that will support PHP 5.3.
```
Comment (Ocramius) [inline site/2015/03/31/doctrine_mongo_odm_module_release_0_8_2.rst]: `` instead of `

File: site/master.rst [hunk 3]
```diff
@@ -4,6 +4,7 @@ Sitemap
 .. toctree::
    :maxdepth: 1
 
+   2015/03/31/doctrine_mongo_odm_module_release_0_8_2
```
Comment (Ocramius) [inline site/master.rst]: Please use dashes, not underscores ;-)

**Comment by samsonasik** [comment]
need rebase

**Comment by gianarb** [comment]
:+1:

**Comment by gianarb** [comment]
Thanks @beberlei

**Pull request merged**